### PR TITLE
Hw1 task2

### PR DESCRIPTION
**Pull request by matveyprupas**
Task 2 from HW1

Task assignment https://epam.sharepoint.com/sites/LMSO/Program/318459c0b61e-edc9-1d99-e518-4606666a0fdd/Scope,%20Hoisting,%20Closures.pdf

### CODE REVIEW

**Review by NikitaSpirit — COMMENTED** [review]
@matveyprupas Please, include link to document with task to the description

**Comment by matveyprupas** [comment]
> @matveyprupas Please, include link to document with task to the description

Done

File: hw1_task2/index.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+function sum(...args) {
+
+    let sum = args.reduce( (sum, e) => sum + e );
```
Comment (NikitaSpirit) [inline hw1_task2/index.js]: @matveyprupas We run the same reduce function twice. Could we fix that?

**Review by NikitaSpirit — CHANGES_REQUESTED** [review]

File: hw1_task2/index.js (hunk 1 above)
Comment (matveyprupas) [reply hw1_task2/index.js]: We can fix it by using a little function for this operation. But it will be the same effectivity.

**Comment by matveyprupas** [comment]
@NikitaSpirit Fixed

**Review by NikitaSpirit — APPROVED** [review]

**Comment by NikitaSpirit** [comment]
Resolve conflicting files and merge it :)

**Pull request merged**